centreon/tests/e2e/features/Api-Token/03-api-token-sorting.feature

```diff
@@ -0,0 +1,35 @@
+@ignore
+@REQ_MON24868
+Feature: Sorting API Tokens List
+
+  As an administrator
```

Comment:
Isn t there a way to make less scenarios by using the Scenario outline / example table since the core actions are always the same = click on column header and output is "sorted" ?
This would optimize the tests duration.